### ISSUE TITLE
Param "Enemy uses Anti-Air" question

### ISSUE THREAD

**Issue by Doctor-Butts** [Vdauphin/HeartsAndMinds#403]
What anti-air units does this refer to? Vehicle based anti-air (Shilka), soldier-equipped anti-air (igla), or both?

**Comment by Vdauphin** [comment]
Hello,
This mean the H&M will looking for soldier-equipped anti-air (igla). https://github.com/Vdauphin/HeartsAndMinds/blob/master_stable/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/mil/class.sqf#L60-L63

Do you have any suggestion to avoid this mistake ?

Bye

**Comment by Doctor-Butts** [comment]
Hi,

Thanks for the clarification. I'd only ask that maybe in an updated version, reword it to specify that it will not spawn Anti-Air infantry. Thanks for the option, though, we've found that Iglas are pretty tough and it's easier to disable it this way than editing a spawn list of troops.
